core/src/main/scala/com/banno/kafka/consumer/ConsumerOps.scala

```diff
@@ -33,16 +33,39 @@ import com.banno.kafka._
 import fs2.concurrent.{Signal, SignallingRef}
 import org.typelevel.log4cats.slf4j.Slf4jLogger
 
-sealed trait SeekTo {
-  def apply[F[_]](consumer: ConsumerApi[F, ?, ?], partitions: Iterable[TopicPartition]): F[Unit]
-}
-case object SeekToBeginning extends SeekTo {
-  def apply[F[_]](consumer: ConsumerApi[F, ?, ?], partitions: Iterable[TopicPartition]): F[Unit] =
-    consumer.seekToBeginning(partitions)
-}
-case object SeekToEnd extends SeekTo {
-  def apply[F[_]](consumer: ConsumerApi[F, ?, ?], partitions: Iterable[TopicPartition]): F[Unit] =
-    consumer.seekToEnd(partitions)
+sealed trait SeekTo
```

Comment:
This is a breaking change. Do we not expect breakage in practice?

Reply:
No code changes will be needed, only recompiling.

Reply:
It's not immediately obvious to me why that is true. Let's talk about it on the huddle tomorrow.